Airline_Web_API/Airline_Web_API/Startup.cs

```diff
@@ -29,6 +29,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDbContext<AirlineContext>(options
                => options.UseSqlServer(Configuration.GetConnectionString("AirlineDatabase")));
             services.AddControllers();
+            services.AddControllersWithViews()
+                    .AddNewtonsoftJson(options =>
+                    options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore
```

Comment:
Why do you need this?

Reply:
I need it to allow converting tickets array and airports in returned flight to JSON